tests/h/views/admin/users_test.py

```diff
@@ -233,22 +233,35 @@ def models(patch):
 
 @pytest.fixture
 def user_service(pyramid_config, db_session):
-    service = Mock(spec_set=UserService(default_authority='example.com',
-                                        session=db_session))
+    service = mock.create_autospec(
```

Comment:
These changes look fine, but they appear to be unrelated to the purpose of the PR? It's always fine to submit small cleanup PRs separately - whether they relate to the code you are currently working on or not.